Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -514,6 +544,15 @@ void ABenchmarkGymGameModeBase::ParsePassedValues()
 				{
 					NumWorkers = FCString::Atoi(*NumWorkersString);
 				}
+
+				FString StatStartProfileDelaySring, StatStopProfileDelayString;
+				if (SpatialWorkerFlags->GetWorkerFlag(NumWorkersWorkerFlag, StatStartProfileDelaySring) && SpatialWorkerFlags->GetWorkerFlag(NumWorkersWorkerFlag, StatStopProfileDelayString))
```

Comment:
I would prefer separating this into two conditions, using a reasonable default for stop profiling (e.g. 60 seconds), generally speaking I believe that flags should work individually.

Reply:
stat startfile and stopfile should happen in pairs. 
So, I will change it to stat_profile: begin, duration